cloudera-director-on-centos/scripts/azure.simple.conf

```diff
@@ -93,10 +93,10 @@ instances {
 
         #
         # VM type.
-        # NOTE: Per Azure RA, only DS13 & DS14 are supported.
+        # NOTE: See Azure RA http://tiny.cloudera.com/azure-ra for recommended VM types.
         #
 
-        type: STANDARD_DS14
+        type: STANDARD_DS14_v2
```

Comment:
I know this was the way it was before your changes - but what happens if these don't match what's used in the template?  IOW, can/should we pass these values in rather than hardcode?

Reply:
This change sets the default value for this config file that customers can reference.

We do pass the values in and generate a new config file `azure.simple.expanded.conf` from all the fields in the template - see https://github.com/Azure/azure-quickstart-templates/blob/6cedb2c090855f52b6d50f97395fdf4295a4517c/cloudera-director-on-centos/scripts/prepare-director-conf.py#L95